### PR TITLE
BE-038-2 (Nikolay) Add update (Locations)

### PR DESCRIPTION
**Pull request by mblack666**

### CODE REVIEW

File: src/main/java/com/exadel/discount/controller/VendorLocationController.java [hunk 1]
```diff
@@ -51,6 +53,14 @@ public LocationDTO addVendorLocation(@RequestBody @Valid CreateLocationDTO vendo
         return vendorLocationService.save(vendorLocation);
     }
 
+    @PutMapping("/{id}")
+    @AdminAccess
+    @ApiOperation("Edit location by ID")
+    public LocationDTO updateLocation(@PathVariable @NotNull UUID id,
+                                  @RequestBody @NotNull UpdateLocationDTO locationDTO) {
```
Comment (Hanna-Kalashnyk) [inline src/main/java/com/exadel/discount/controller/VendorLocationController.java]: Alt+Ctrl+L, please

File: src/main/java/com/exadel/discount/service/impl/VendorLocationServiceImpl.java [hunk 2]
```diff
@@ -31,14 +33,8 @@
     @Override
```
Comment (Hanna-Kalashnyk) [inline src/main/java/com/exadel/discount/service/impl/VendorLocationServiceImpl.java:33]: @Transactional?

**Review by Hanna-Kalashnyk — DISMISSED** [review]

File: src/main/java/com/exadel/discount/service/impl/VendorLocationServiceImpl.java (hunk 2 above)
Comment (mblack666) [reply src/main/java/com/exadel/discount/service/impl/VendorLocationServiceImpl.java:33]: its work well without @transactional

File: src/main/java/com/exadel/discount/controller/VendorLocationController.java (hunk 1 above)
Comment (mblack666) [reply src/main/java/com/exadel/discount/controller/VendorLocationController.java]: +

**Review by Hanna-Kalashnyk — APPROVED** [review]

**Review by rudik-byte — APPROVED** [review]

**Pull request merged**